### PR TITLE
Observer validator table

### PR DESCRIPTION
**Pull request by fadeev**

### CODE REVIEW

File: docs/architecture/observers.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 9
+---
+
+import ObserverList from "@site/src/components/ObserverList/ObserverList";
+import ObserverParams from "@site/src/components/ObserverParams/ObserverParams";
+
+# Observers
```
Comment (btroo) [inline docs/architecture/observers.mdx:8]: I think since they are coupled, could we call it Observer-Signer throughout this page?

**Review by btroo — COMMENTED** [review]
Just one naming comment to resolve. This is super helpful to have!

**Comment by fadeev** [comment]
Updated to "observer-signers". Didn't change the component names as it's not user visible and we'll have separate components for fetching signers data when that becomes relevant.

**Review by bbbeeeee — COMMENTED** [review]
Apologies for dropping the ball on this. Could we make sure this also shows the mainnet beta network? Otherwise, looks good to me.

**Comment by fadeev** [comment]
@bbbeeeee made compatible with mainnet beta.

**Review by bbbeeeee — APPROVED** [review]

**Pull request merged**